src/spikeinterface/widgets/motion.py

```diff
@@ -96,55 +94,241 @@ def plot_matplotlib(self, data_plot, **backend_kwargs):
+            times=times,
+            sampling_frequency=sampling_frequency,
+            segment_index=segment_index,
+            depth_lim=depth_lim,
+            color_amplitude=color_amplitude,
+            color=color,
+            scatter_decimate=scatter_decimate,
+            cmap=cmap,
+            clim=clim,
+            alpha=alpha,
+            recording=recording,
+        )
+        BaseWidget.__init__(self, plot_data, backend=backend, **backend_kwargs)
+
+    def plot_matplotlib(self, data_plot, **backend_kwargs):
+        import matplotlib.pyplot as plt
+        from matplotlib.colors import Normalize
+        from .utils_matplotlib import make_mpl_figure
+
+        from spikeinterface.sortingcomponents.motion_interpolation import correct_motion_on_peaks
+
+        dp = to_attr(data_plot)
+
+        assert backend_kwargs["axes"] is None, "axes argument is not allowed in DriftRasterMapWidget. Use ax instead."
+
+        self.figure, self.axes, self.ax = make_mpl_figure(**backend_kwargs)
+
+        if dp.times is None:
+            peak_times = dp.peaks["sample_index"] / dp.sampling_frequency
+        else:
+            peak_times = dp.times[dp.peaks["sample_index"]]
+
+        peak_locs = dp.peak_locations[dp.direction]
+        if dp.scatter_decimate is not None:
+            peak_times = peak_times[:: dp.scatter_decimate]
+            peak_locs = peak_locs[:: dp.scatter_decimate]
+
+        if dp.color_amplitude:
+            amps = dp.peak_amplitudes
+            amps_abs = np.abs(amps)
+            q_95 = np.quantile(amps_abs, 0.95)
+            if dp.scatter_decimate is not None:
+                amps = amps[:: dp.scatter_decimate]
+                amps_abs = amps_abs[:: dp.scatter_decimate]
+            cmap = plt.colormaps[dp.cmap]
+            if dp.clim is None:
+                amps = amps_abs
+                amps /= q_95
+                c = cmap(amps)
+            else:
+                norm_function = Normalize(vmin=dp.clim[0], vmax=dp.clim[1], clip=True)
+                c = cmap(norm_function(amps))
+            color_kwargs = dict(
+                color=None,
+                c=c,
+                alpha=dp.alpha,
+            )
+        else:
+            color_kwargs = dict(color=dp.color, c=None, alpha=dp.alpha)
+
+        self.ax.scatter(peak_times, peak_locs, s=1, **color_kwargs)
+        if dp.depth_lim is not None:
+            self.ax.set_ylim(*dp.depth_lim)
+        self.ax.set_title("Peak depth")
+        self.ax.set_xlabel("Times [s]")
+        self.ax.set_ylabel("Depth [$\\mu$m]")
+
+
 class MotionInfoWidget(BaseWidget):
     """
-        Plot motion information from the motion_info dict returned by correct_motion().
-        This plot:
-          * the motion iself
-          * the peak depth vs time before correction
-          * the peak depth vs time after correction
+    Plot motion information from the motion_info dictionary returned by the `correct_motion()` funciton.
+    This widget plots::
```

Comment:
```suggestion
    This widget plots:
```

did we need the two : ?